services/core-web/common/constants/reducerTypes.js

```diff
@@ -122,6 +122,19 @@ export const GET_VARIANCE_DOCUMENT_CATEGORY_OPTIONS = "GET_VARIANCE_DOCUMENT_CAT
 export const UPDATE_MINE_VARIANCE = "UPDATE_MINE_VARIANCE";
 export const DELETE_VARIANCE = "DELETE_VARIANCE";
 
+// Project Summaries
+export const PROJECT_SUMMARIES = "PROJECT_SUMMARIES";
+export const CREATE_MINE_PROJECT_SUMMARY = "CREATE_MINE_PROJECT_SUMMARY";
+export const GET_PROJECT_SUMMARIES = "GET_PROJECT_SUMMARIES";
+export const GET_PROJECT_SUMMARY = "GET_PROJECT_SUMMARY";
+export const ADD_DOCUMENT_TO_PROJECT_SUMMARY = "ADD_DOCUMENT_TO_PROJECT_SUMMARY";
+export const REMOVE_DOCUMENT_FROM_PROJECT_SUMMARY = "REMOVE_DOCUMENT_FROM_PROJECT_SUMMARY";
+export const GET_PROJECT_SUMMARY_STATUS_OPTIONS = "GET_PROJECT_SUMMARY_STATUS_OPTIONS";
+export const GET_PROJECT_SUMMARY_DOCUMENT_CATEGORY_OPTIONS =
```

Comment:
"GET_PROJECT_SUMMARY_STATUS_OPTIONS" this doesn't look like its being used anywhere